screen.css

```diff
@@ -0,0 +1,41 @@
+.table{
+    border:4px solid blue;
+    width:100%;
```

Comment:
font size is not good

Reply:
all the font is in the 50px 


should I change the font size

Reply:
Yes..make it around 30px and It will look good..
@dpnraju95